tests/sparseml/transformers/utils/test_helpers.py

```diff
@@ -170,26 +170,43 @@ def test_save_zoo_directory(tmp_path, stub):
 
 
 @pytest.mark.parametrize(
-    "string, prompt, censor, expected_mask",
+    "string, response, prompt, expected_mask",
     [
-        ("[foo]hello\n\n[bar]world", "[foo]", "[bar]", "1111111111110000000000"),
+        (
+            ("[foo]hello\n\n" "[bar]world"),
+            "[bar]",
+            "[foo]",
+            ("000000000000" "1111111111"),
```

Comment:
Do we support a situation where we don't start with a prompt? Ie a mask like "111100000111110000"? If so can we add another unit test case for that, or if not note this in the `generate_mask` function docstring?